core/internal/cltest/cltest.go

```diff
@@ -1397,6 +1399,13 @@ func AssertPipelineRunsStays(t testing.TB, pipelineSpecID int32, store *strpkg.S
 	return prs
 }
 
+func AssertPipelineTaskRunsSuccessful(t testing.TB, runs []pipeline.TaskRun) {
+	t.Helper()
+	for _, run := range runs {
+		require.True(t, run.Error.IsZero())
```

Comment:
`require.NoError` would be better here, no?

Reply:
I'd also suggest maybe adding a message to indicate which of the pipeline task runs failed, including the index in the array, the name, etc.

Reply:
Normally yeah, but this is not actually a Go `error` type -- it's a `null.String`